azure-pipelines.yml

```diff
@@ -19,276 +19,26 @@ jobs:
-
-      - task: PublishBuildArtifacts@1
-        condition: and(
-          in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-          ne(variables['Build.Reason'], 'PullRequest'))
-        inputs:
-          PathtoPublish: './common/temp/bundleAnalysis'
-          ArtifactName: '$(bundleArtifactName)'
-        displayName: 'Publish bundle analysis'
-
-      - powershell: |
-          $npmVer=$(node -p "require('./packages/teams-js/package.json').version")
-          Write-Host "##vso[task.setvariable variable=version;isOutput=true]$npmVer"
-        name: package
-        displayName: 'Set package.version Variable'
-
-      - task: CopyFiles@2
-        inputs:
-          sourceFolder: 'apps/teams-test-app/build'
-          contents: '**'
-          targetFolder: '$(Build.ArtifactStagingDirectory)\teams-test-app'
-        displayName: 'Copy Test app to artifacts staging directory'
-
-      - task: ArchiveFiles@2
-        inputs:
-          rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\teams-test-app'
-          includeRootFolder: false
-          archiveType: 'zip'
-          archiveFile: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-          replaceExistingArchive: true
-        displayName: 'Zip Test app artifacts'
-
-      - task: PublishBuildArtifacts@1
-        inputs:
-          PathtoPublish: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-          ArtifactName: 'teams-test-app'
-        displayName: 'Publish Test app artifacts'
-
-      - task: CopyFiles@2
-        inputs:
-          sourceFolder: 'packages\teams-js\dist'
-          contents: '**\?(*.js|*.ts|*.map)'
-          targetFolder: '$(Build.ArtifactStagingDirectory)\CDNFeed\$(package.version)\js'
-        displayName: 'Copy TeamsJS Content for CDN'
-
-      - task: CopyFiles@2
-        inputs:
-          Contents: |
-            packages\teams-js\package.json
-            packages\teams-js\README.md
-            LICENSE
-          TargetFolder: '$(Build.ArtifactStagingDirectory)\NPMFeed'
-          flattenFolders: true
-        displayName: 'Copy TeamsJS Content for NPM'
-
-      - task: CopyFiles@2
-        inputs:
-          Contents: |
-            packages\teams-js\dist\**\?(*.js|*.ts|*.map)
-          TargetFolder: '$(Build.ArtifactStagingDirectory)\NPMFeed\dist'
-          flattenFolders: true
-        displayName: 'Copy JS Content for NPM'
-
-      - task: PublishBuildArtifacts@1
-        inputs:
-          PathtoPublish: '$(Build.ArtifactStagingDirectory)\CDNFeed'
-          ArtifactName: 'CDNFeed'
-        displayName: 'Publish CDN feed to build Artifacts'
-
-      - task: PublishBuildArtifacts@1
-        inputs:
-          PathtoPublish: '$(Build.ArtifactStagingDirectory)\NPMFeed'
-          ArtifactName: 'NPMFeed'
-        displayName: 'Publish NPM feed to Build Artifacts'
-
-  - job: E2e_Test
-    displayName: 'E2e test'
+  - job: E2ETest1
```

Comment:
Does the order the jobs are listed in have any impact on which tests start (or finish) first?

Reply:
I can answer this one: Unless there is a dependsOn/conditional property, the jobs are run in parallel.